backend/src/api/app.ts

```diff
@@ -38,7 +38,7 @@ const handlerToExpress = (handler) => async (req, res, next) => {
     res.status(statusCode).json(parsedBody);
   } catch (e) {
     // Not a JSON body
-    res.status(statusCode).send(body);
+    res.status(statusCode).json({ resp: body });
```

Comment:
Actually, this doesn't work, because the logs endpoint returns plain text, not JSON.

Reply:
ah - I'll just make it respond with text/plain then instead of JSON